schemas/dev/observation.schema.json

```diff
@@ -124,6 +138,7 @@
         "joint_pain",
         "loss_of_smell",
         "loss_of_taste",
+        "loss_of_smell_or_taste",
```

Comment:
nice add! i think one of the recent had taste and smell combined into one field ..